portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -81,21 +93,18 @@ private String getParameter(HttpServletRequest request, String name) {
     return value;
   }
 
-  private int getMaxCommentsToDisplay(HttpServletRequest request) throws IllegalArgumentException {
-    String maxCommentsToDisplayString = "3"; //temporary hardcoded value, will get replaced with user input
-
+  private void setMaxCommentsToDisplay(HttpServletRequest request) throws IllegalArgumentException {
+    String maxCommentsToDisplayString = getParameter(request, MAX_COMMENTS_QUERY_STRING);
     // Convert the input to an int.
-    int maxCommentsToDisplay;
     try {
         maxCommentsToDisplay = Integer.parseInt(maxCommentsToDisplayString);
```

Comment:
You're parsing this as an Int. The html form allows me enter numbers without an error.

If I enter a number greater than MAX_INT it sets the number of comments to 5, which seems non-intuitive.

Reply:
I changed it so that the HTML form has a max a 100 comments. I'm a bit confused on what the change requested is in the first part of the comment. Is the parsing as an int redudant?

Reply:
would it not be best to keep a default value and update the comments based on the user going back and forth

Reply:
Addressed in person